pkg/service/test/match.go

```diff
@@ -72,7 +71,11 @@ func jsonMatch(key string, expected, actual interface{}, noiseMap map[string]boo
 	}
 	switch x.Kind() {
 	case reflect.Float64, reflect.String, reflect.Bool:
-		if expected != actual && !noiseMap[key] {
+		_, ok := noiseMap[key]
+		if ok && len(noiseMap[key]) != 0 {
+			ok = MatchesAnyRegex(actual.(string), noiseMap[key])
```

Comment:
You can also give option to write field name as regex along with value regex which you implemented. but field regex will not be array of strings it will be just string.

Reply:
done!